docs/domains/Domain.json

```diff
@@ -183,6 +183,14 @@
           "description": "Model in image model files and properties.",
           "$ref": "#/definitions/Model"
         },
+        "distributionStrategy": {
```

Comment:
Can we consider changing the name to 'introspectVersionDistributionStrategy' or 'situationalConfigurationDistributionStrategy'?   The distribution strategy term is very generic - it doesn't let you know what the field is meant for.  Also the generic nature introduces potential conflicts - for example, we'll likely have a 'modelDistributionStrategy' at some point.

Reply:
It's under the `configuration` element. That should give it the relevant context. If it were at the spec level, I'd agree with you.

Reply:
I disagree. The configuration element is getting crowded - there are attributes that have nothing to do with the new setting.  And there are attribute(s) that may have there own 'strategy' setting soon enough.

Reply:
After some discussion:  'overridesDistributionStrategy'.